tests/test_meshtastic_ble.py

```
@@ -193,6 +272,130 @@ def test_exception_returns_true(self):
+        iface.client.bleak_client.address = "22:33:44:55:66:77"
+
+        assert _extract_ble_address_from_interface(iface) == "223344556677"
+
+    def test_ignores_property_access_errors_and_continues(self):
+        from mmrelay.meshtastic.ble import _extract_ble_address_from_interface
+
+        class RaisingProperty:
+            def __init__(self, exc: Exception) -> None:
+                self._exc = exc
+
+            def __get__(self, obj: object, type: type | None = None) -> NoReturn:
```

Ruff (0.15.11)

[error] Function argument `type` is shadowing a Python builtin

(A002)